### ISSUE TITLE
Node auto expansion

### ISSUE THREAD

**Issue by cesek-navisys** [d-kimuson/ts-type-expand#337]
Add some options to allow users to setup auto expansion or at least expand the first level automatically.

**Comment by dkstrong** [comment]
would also find this super useful. Usually the reason I'm trying to expand and view the types like this is to navigate really complex types. It would be great to fully expand the structure by default.

**Comment by TonyGravagno** [comment]
👍

**Comment by GitMurf** [comment]
Just installed this extension and love it so far... but this was the exact reason I came to GH to raise an issue / question about expanding the types by default. Otherwise I am constantly having to mouse over and expand when I really just want an ongoing view of the active type without needing to expand each time manually.

**Comment by GitMurf** [comment]
@senkenn this is wonderful! Do you know if it is possible to use your version of the plugin as beta until the PR gets accepted here?

**Comment by senkenn** [comment]
> @senkenn this is wonderful! Do you know if it is possible to use your version of the plugin as beta until the PR gets accepted here?

Yeah, you can install my version by following these steps
1. Clone my forked repo: https://github.com/senkenn/ts-type-expand/tree/feature/issue%23337
2. With Node.js and Yarn installed, create a VSIX file
   ```ts
   npm install -g @vscode/vsce
   cd packages/ts-type-expand && vsce package --yarn
   ```
   The VSIX file will be generated under `packages/ts-type-expand`
3. In command palette, enter this command: `Extensions: Install from VSIX...` and select the VSIX file

**Comment by GitMurf** [comment]
Thank you @senkenn!

**Comment by d-kimuson** [comment]
We have just released v1.0.6 which incorporates these changes.
You can update and try it out at your end.
Thank you very much!

https://github.com/d-kimuson/ts-type-expand/releases/tag/v1.0.6
